src/ert/__main__.py

```diff
@@ -528,15 +528,24 @@ def get_ert_parser(parser: Optional[ArgumentParser] = None) -> ArgumentParser:
         type=valid_name,
         default=None,
         action=DeprecatedAction,
-        dest="restart_ensemble",
+        dest="restart_ensemble_id",
         help="Deprecated: This argument is deprecated and will be "
         "removed in future versions. Use --restart-ensemble instead.",
     )
     es_mda_parser.add_argument(
         "--restart-ensemble",
         type=valid_name,
         default=None,
-        help="Name of the ensemble where the results for the experiment "
+        dest="restart_ensemble_id",
+        help="Deprecated: This argument is deprecated and will be "
+        "removed in future versions. Use --restart-ensemble-id instead.",
```

Comment:
Are we sure we want users to specify an `id` instead of a name, as it's tricky to find the `id` of ensembles.

Reply:
I agree the uuids for an ensemble is hard to find. Discussed this with @oyvindeide and we agreed that this change is ok as it is in a setting that is used by more advanced users